Makefile

```diff
@@ -14,7 +14,6 @@ build:
 	@echo "Building images for tags: $(TAGS)"
 	set -e; for i in $(TAGS); do printf "\nBuilding $(NAME):$$i \n\n"; cd php$$i; \
 		DOCKER_BUILDKIT=$(DOCKER_BUILDKIT) docker build -t $(NAME):$$i \
-		--platform $(PLATFORM) \
```

Comment:
this option is available because `DOCKER_BUILDKIT=1` means that build is using `buildx`

Reply:
From my experience, simple `build` is like `buildx build --load` which doesn't provide multiarch.
BTW, since Docker 23 it's always buildkit, but it still doesn't support multiarch for `build`

Reply:
I mean we just need to remove usage of `DOCKER_BUILDKIT`